test/test_journalpump.py

```diff
@@ -1,9 +1,14 @@
+import json
```

Comment:
Alphabetically this would come after the from's

Reply:
fixed